src/shared/utils/mixpanel/mixpanel.utils.ts

```diff
@@ -27,40 +27,98 @@ export const trackAppletSave = ({
 }) => {
   if (!applet) return;
 
-  const itemTypes: ItemResponseType[] = [];
+  let itemCount = 0;
+  let phraseBuilderItemCount = 0;
+  let itemsIncludedInPhraseBuilders = 0;
+  const phraseBuilderFieldCounts = {
+    itemResponse: 0,
+    sentence: 0,
+    lineBreak: 0,
+  };
   let autoAssignedActivityCount = 0;
   let manualAssignedActivityCount = 0;
-  for (const activity of applet.activities ?? []) {
-    itemTypes.push(...(activity.items ?? []).map((item) => item.responseType));
+  const itemTypes: ItemResponseType[] = [];
+
+  for (const activity of applet.activities) {
     if (activity.autoAssign) {
       autoAssignedActivityCount++;
     } else {
       manualAssignedActivityCount++;
     }
+
+    const visibleItemNames = new Set<string>();
+    const referencedItemNames = new Set<string>();
+    const items = activity.items;
+
+    for (const item of items) {
+      // Hidden items are not included in any counts
+      if (item.isHidden) continue;
+
+      itemCount++;
+      visibleItemNames.add(item.name);
+      itemTypes.push(item.responseType);
+
+      if (item.responseType === ItemResponseType.PhrasalTemplate) {
+        phraseBuilderItemCount++;
+        const referencedItemNames = new Set<string>();
+
+        for (const phrase of item.responseValues.phrases) {
+          for (const field of phrase.fields) {
+            if (field.type === 'item_response') {
+              if (visibleItemNames.has(field.itemName)) {
+                referencedItemNames.add(field.itemName);
+                phraseBuilderFieldCounts.itemResponse++;
+              }
+            } else if (field.type === 'sentence') {
+              phraseBuilderFieldCounts.sentence++;
+            } else if (field.type === 'line_break') {
+              phraseBuilderFieldCounts.lineBreak++;
+            }
+          }
+        }
+        itemsIncludedInPhraseBuilders += referencedItemNames.size;
```

Comment:
What's being counted here is the _sum of the counts of the unique items referenced within each Phrase
Builder_, like you mention in f21c30e0. I trust that this is what is should actually be, but it's a little different from what the ticket description says

> Total count of unique items that were included as Item Response fields in Phrase Builder items in the applet

I can see that being interpreted as what you had implemented before: _count of the unique items referenced across all Phrase Builders_

So we should probably update the ticket description.

Moreover, I think calling this `Items Included in Phrase Builders` is misleading. Given what it actually is, something like `Phrase Builder Unique Item Response Count` (or something like that) is more appropriate

Reply:
I've updated the ticket description to redefine that parameter, and renamed it as suggested!